src/data/NewStaticPageData.json

```diff
@@ -0,0 +1,40 @@
+  "description": [
+    {
+      "header": "What to add",
+      "textBefore": " FAIRsharing maintains three resource registries covering the life and biomedical sciences, that are progressively interrelated: ",
+      "bulletPoints": [
+        "<b>Standards,</b> including (but not limited to) reporting guidelines, terminology artefacts, models and formats, and metrics.",
+        "<b>Databases,</b> focusing on global and community-driven repositories",
+        "<b>Policies,</b> highlighting data preservation, management and sharing"
+      ],
+      "textAfter": " To allow the grouping of records across these three registries, we support the generation of Collections. We encourage Collections collated according to a specific criteria, for example, those databases and standards recommended by a funding body or journal policy. These can be browsed by clicking on the Collections button in the navigation bar above. "
+    },
+    {
+      "header": "Adding Database records",
+      "textBefore": "FAIRsharing has a focus on global and community-driven repositories and generally does not store records describing individual data sets. Although this list is neither required nor exhaustive, databases listed within FAIRsharing generally conform to most of the following:",
+      "bulletPoints": [
+        "Allows viewing, searching and download of the database.",
+        "Serves data from multiple experiments to the community.",
+        "Produces regular releases.",
+        "Allows data submissions from the community.",
+        "Provides documentation on / help with the resource."
+      ],
+      "textAfter": "If your resource is a data set, it may not suitable for FAIRsharing. However, there are other sites you may wish to consider.\n\nIf you are looking for a centralized location to store your data set, <a href='https://fairsharing.org/biodbcore-000470' target='_blank' class='underline-effect'>ClinicalTrials.gov</a> is a registry and results database of publicly and privately supported clinical studies of human participants conducted around the world. Alternatively, the <a href='https://fairsharing.org/' target='_blank' class='underline-effect'>Dryad Digital Repository</a> is a curated resource that makes the data underlying scientific publications discoverable, freely reusable, and citable. Dryad provides a general-purpose home for a wide diversity of datatypes. Data associated with any published article can be uploaded."
```

Comment:
```suggestion
      "textAfter": "If your resource is a data set, it may not suitable for FAIRsharing. However, there are other sites you may wish to consider.\n\nIf you are looking for a centralized location to store your data set, <a href='https://fairsharing.org/biodbcore-000470' target='_blank' class='underline-effect'>ClinicalTrials.gov</a> is a registry and results database of publicly and privately supported clinical studies of human participants conducted around the world. Alternatively, the <a href='https://datadryad.org/stash' target='_blank' class='underline-effect'>Dryad Digital Repository</a> is a curated resource that makes the data underlying scientific publications discoverable, freely reusable, and citable. Dryad provides a general-purpose home for a wide diversity of datatypes. Data associated with any published article can be uploaded."
```